sqlglot/dialects/postgres.py

```diff
@@ -742,3 +742,12 @@ def isascii_sql(self, expression: exp.IsAscii) -> str:
         @unsupported_args("this")
         def currentschema_sql(self, expression: exp.CurrentSchema) -> str:
             return "CURRENT_SCHEMA"
+
+        def interval_sql(self, expression: exp.Interval) -> str:
+            unit = expression.text("unit").lower()
+
+            if unit.startswith("quarter") and isinstance(expression.this, exp.Literal):
+                expression.this.replace(exp.Literal.number(int(expression.this.to_py()) * 3))
+                expression.args["unit"].replace(exp.var("MONTH"))
```

Comment:
Should be a low risk change given that Postgres doesn't support the "quarter" unit in its intervals to begin with:

```
georgesittas=# SELECT INTERVAL '1 QUARTER';
ERROR:  invalid input syntax for type interval: "1 QUARTER"
LINE 1: SELECT INTERVAL '1 QUARTER';
                        ^
georgesittas=# SELECT INTERVAL '1 MONTH';
 interval
----------
 1 mon
(1 row)
```